### PR TITLE
Feat: Adapter, Mole [using uni v3 helper]

### PR DESCRIPTION
**Pull request by 0xpeluche**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/mole exports TVL: 
        
 
 
    sui                       1.60 M
    aptos                     1.71 k
    
    total                    1.60 M

File: projects/mole/utils.js [hunk 1]
```diff
@@ -7,6 +10,198 @@ function i32BitsToNumber(v) {
     return asIntN(BigInt(v), 32);
 }
 
+function toX64_Decimal(num) {
```
[the remark this answers is not in the document]
Comment (0xpeluche) [reply projects/mole/utils.js:13]: yea utils.js file isnt use anymore, do you want me to delete it ?

[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/mole/utils.js:13]: yeah

**Comment by llamatester** [comment]
The adapter at projects/mole exports TVL: 
        
 
 
    sui                       1.68 M
    aptos                     1.75 k
    
    total                    1.68 M

**Pull request merged**